tensorboard/pip_package/requirements.txt

```diff
@@ -24,7 +24,7 @@ google-auth >= 1.6.3, < 3
 google-auth-oauthlib >= 0.4.1, < 0.5
 markdown >= 2.6.8
 numpy >= 1.12.0
-protobuf >= 3.6.0
+protobuf >= 3.12.0
```

Comment:
Might as well just make this 3.9.2 - no reason to be more strict than necessary.

Reply:
Done.